app/routers/event.py

```diff
@@ -25,16 +27,66 @@ async def eventedit(request: Request):
                                       {"request": request})
 
 
-@router.get("/view/{id}")
-async def eventview(request: Request, id: int):
+@router.get("/view/{event_id}")
+async def eventview(request: Request, event_id: int,
+                    db: Session = Depends(get_db)):
+    try:
+        event = get_event_by_id(db, event_id)
+    except NoResultFound:
+        raise HTTPException(status_code=404, detail="Event not found")
+    except MultipleResultsFound:
+        raise HTTPException(status_code=500, detail="Multiple events found")
+    start_format = '%A, %d/%m/%Y %H:%M'
+    end_format = ('%H:%M' if event.start.date() == event.end.date()
+                  else start_format)
     return templates.TemplateResponse("event/eventview.html",
-                                      {"request": request, "event_id": id})
+                                      {"request": request, "event": event,
+                                       "start_format": start_format,
+                                       "end_format": end_format})
+
 
+@router.delete("/{event_id}")
+def delete_event(event_id: int,
+                 db: Session = Depends(get_db)):
+    # TODO: Check if the user is the owner of the event.
+    try:
+        event = get_event_by_id(db, event_id)
+    except NoResultFound:
+        raise HTTPException(status_code=404, detail="Event not found")
+    except MultipleResultsFound:
+        raise HTTPException(status_code=500, detail="Multiple events found")
+
+    participants = get_participants_emails_by_event(db, event_id)
+
+    db.delete(event)  # Delete event
+    db.query(UserEvent).filter_by(event_id=event_id) \
```

Comment:
Prefer not to use backslash for the split. In this case, you can merge the lines. (You can remove the comment)